### PR TITLE
Skip asking for region when there's only a single option

### PR DESCRIPTION
**Pull request by pgilmorepf**
Utility method to encapsulate region picking
Skip asking for region when there's only a single option
Default VS auto-spacing

### CODE REVIEW

File: MpsAllocatorSample/Program.cs [hunk 1]
```diff
@@ -114,17 +114,21 @@ static async Task ListBuildSummaries()
...
+
         static async Task ListMultiplayerServers()
         {
             var req = new PlayFab.MultiplayerModels.ListMultiplayerServersRequest();
             string buildID = ReadBuildIDFromInput();
             var regions = await GetRegions(buildID);
-            Console.WriteLine($"Enter region (options are {string.Join(",", regions)})");
-            string region = Console.ReadLine();
+            string region = regions.FirstOrDefault();
+            if (regions.Count() > 1)
```
Comment (dgkanatsios) [inline MpsAllocatorSample/Program.cs:127]: are we auto-picking a region here? If so, maybe we can tell the user (Console.WriteLine) that we are using a specific region

Comment (dgkanatsios) [reply MpsAllocatorSample/Program.cs:127]: also, can we have that change specified in the PR description so it's better tracked?

**Review by dgkanatsios — APPROVED** [review]

File: MpsAllocatorSample/Program.cs [hunk 2]
```diff
@@ -234,9 +228,7 @@ static async Task PrintServerStateInVM()
             var req = new PlayFab.MultiplayerModels.ListMultiplayerServersRequest();
             string buildID = ReadBuildIDFromInput();
             var regions = await GetRegions(buildID);
```
Comment (pedrofv) [inline MpsAllocatorSample/Program.cs:230]: >await GetRegions(buildID); [](http://example.com/codeflow?start=26&length=26)

Nit: 'regions' is only used to be passed as parameter to the other function. Maybe we can just consolidate it inside 'ReadRegionFromInput'.

**Review by pedrofv — APPROVED** [review]
:shipit:

File: MpsAllocatorSample/Program.cs (hunk 1 above)
Comment (pgilmorepf) [reply MpsAllocatorSample/Program.cs]: Yes, and I think I did that in a later commit.

**Pull request merged**